Sources/Workflow/LinkedList/LinkedListMutatingOperations.swift

```diff
@@ -218,7 +218,9 @@ extension LinkedList {
      - Important: This operation mutates the original `LinkedList`.
```

Comment:
Assuming my earlier comment is correct, we should call out that it does *not* mutate the nodes.

Reply:
done